.travis/copyright_mistake_scanner.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+#header checker for either copyright info and/or proper year
+
+YEAR=$(date +%Y)
+while IFS= read -r -d $'\0' file; do
+	grep -L "* Copyright $YEAR" $file | perl -ne 'print "File is missing copyright info: $_"'
+done < <(find . -type f -print0 -name '*.c' -o -name '*.h')
```

Comment:
I think `-print0` should go at the end of the command. On my system the the two `-name` conditionals are ignored when -print0 precedes them https://gist.github.com/raycoll/31eb527a9ceef3866fae3866847c0e98